lms/templates/credit/fake_credit_eligibility_status.html

```diff
@@ -0,0 +1,27 @@
+<%namespace name='static' file='/static_content.html'/>
+<html>
+<head><title>Fake Page to Update the minimum grade course requirement status</title>
+</head>
+<body>
+<p>Fake Credit Eligibility
+<div>
+% if error:
+    <div style="padding-top: 5px;color: red">${ error }</div>
```

Comment:
@awais786 Even if it is a testing page we need to follow standards. If we end up building upon some sort of test page infrastructure, then it would behoove us to use CSS anyway.